package.json

```diff
@@ -77,5 +80,16 @@
       "last 1 safari version"
     ]
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.7",
+    "enzyme": "^3.11.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-import": "^2.26.0",
```

Comment:
are you goint to configure some pre-commit?

Reply:
I juts wanted to try the Linter to improve my code, but at the end it was hard to use, so I just took it out.